### PR TITLE
docs(api): docstring for `load_waste_chute()`

### PR DESCRIPTION
**Pull request by ecormany**
# Overview

API Reference entry for `load_waste_chute()`.

# Test Plan

[Sandbox](http://sandbox.docs.opentrons.com/docstring-load_waste_chute/v2/new_protocol_api.html#opentrons.protocol_api.ProtocolContext.load_waste_chute)

# Changelog

- draft docstring for `load_waste_chute()` behavior
- removed TODOs to confirm product names (they are accurate)

# Review requests

- [ ] final confirmation on `cover` parameter
- [ ] final confirmation on any other parameters

# Risk assessment

v low, docs only

### CODE REVIEW

File: api/src/opentrons/protocol_api/protocol_context.py [hunk 1]
```diff
@@ -439,13 +439,22 @@ def load_adapter_from_definition(
...
-    # TODO: Confirm official naming of "waste chute".
     def load_waste_chute(
         self,
         *,
-        # TODO: Confirm official naming of "staging area slot".
         with_staging_area_slot_d4: bool = False,
     ) -> WasteChute:
+        """Load the waste chute on the deck.
+
+        The waste chute always occupies slot D3.
```
Comment (SyntaxColoring) [inline api/src/opentrons/protocol_api/protocol_context.py]: Terminology nitpick: I think of the waste chute as *replacing* slot D3, not occupying it.

At the internal technical level, this distinction is precise. The D3 *cutout* can have, mounted onto, it a *waste chute cutout fixture,* which provides waste chute capabilities, or a *slot cutout fixture,* which provides slot capabilities. If you have a waste chute, there was never any slot there to begin with.

File: api/src/opentrons/protocol_api/protocol_context.py [hunk 2]
```diff
@@ -439,13 +439,22 @@ def load_adapter_from_definition(
...
+
+        :param bool cover: Set to ``True`` when the cover is attached to the top of
```
Comment (SyntaxColoring) [inline api/src/opentrons/protocol_api/protocol_context.py]: Is *cover* the official public-facing term? I've mostly been using *lid.*

File: api/src/opentrons/protocol_api/protocol_context.py [hunk 3]
```diff
@@ -439,13 +439,22 @@ def load_adapter_from_definition(
...
+            the waste chute. The hole in the cover can fit up to eight tips in a
+            column configuration. Pipettes can dispense, blow out, or drop tips
+            through the hole in the cover. When the cover is attached, the gripper
+            can't drop labware down the chute. If you try to move labware to the
```
Comment (SyntaxColoring) [inline api/src/opentrons/protocol_api/protocol_context.py]: Might be worth a cross-reference to `ProtocolContext.move_labware()`.

**Review by SyntaxColoring — APPROVED** [review]
Looks great, thanks! And thanks for resolving those `# TODO` comments for us.

**EDIT: I'm [questioning now](https://opentrons.slack.com/archives/CSPMY9048/p1701299443179999) whether we even need a `with_cover` arg, so these suggestions might be moot.**

File: api/src/opentrons/protocol_api/protocol_context.py (hunk 2 above)
Comment (jwwojak) [reply api/src/opentrons/protocol_api/protocol_context.py]: Yes, that is correct, "cover" is the term for that piece that fits over the opening. See [le manual](https://insights.opentrons.com/hubfs/Products/Flex/Opentrons%20Flex%20Waste%20Chute%20Quickstart%20Guide.pdf) for the parts list.

File: api/src/opentrons/protocol_api/protocol_context.py (hunk 1 above)
Comment (ecormany) [reply api/src/opentrons/protocol_api/protocol_context.py]: Thinking about the clearest way to phrase this for API users. Let's say you want to put labware down the chute in your protocol. Would your code look like this?
```
chute = protocol.load_waste_chute()
protocol.move_labware(plate, chute, use_gripper=True)
```

Comment (SyntaxColoring) [reply api/src/opentrons/protocol_api/protocol_context.py]: Yep, exactly.

Comment (ecormany) [reply api/src/opentrons/protocol_api/protocol_context.py]: OK, I hit upon it. The _deck plate adapter_ for the waste chute goes in D3. The waste chute goes on top of that. The API is only concerned about the chute (and its addressable areas) but that's the physical reality when you install the chute.

**Comment by ecormany** [comment]
Getting this merged in, assuming that all calls will be with no arguments: `protocol.load_waste_chute()`. Can follow up if that changes.

**Pull request merged**